components/app/landing/LandingJobOffers.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <LandingSectionContainer class="my-32">
      <!-- Headline ------------------------------------------------------------------>
```

Comment:
chore (blocking): this might be outdated but I have not found any other reference. Do not use adjacent hyphens.
Check this article for reference: https://weblog.200ok.com.au/2008/01/dashing-into-trouble-why-html-comments.html